src/main/java/com/epam/test_generator/services/JiraService.java

```diff
@@ -195,14 +212,21 @@ private void updateSuitFromJiraStory(JiraStory jiraStory) {
         if (suit == null) {
             return;
         }
-        suit.setName(jiraStory.getName());
-        suit.setDescription(jiraStory.getDescription());
-        suit.setPriority(getPriority(jiraStory.getPriority()));
-        suit.setLastJiraSyncDate(LocalDateTime.now());
-
         suitDAO.save(suit);
     }
 
+    private Status getBDDStatus(String jiraStatus) {
+        Status status;
```

Comment:
It seems, that you've already added enum JiraStatus.
May be you could use it instead of String?

Reply:
Fixed

Reply:
That is better.
However, may be it is more convenient to use enum in the JiraStory class, so you could hold change the signature of the getBDDStatus method to
--------
private Status getBDDStatus(JiraStatus jiraStatus) {...}
--------
What do you think?

Reply:
Yes, it would be absolutely better. We will change this soon.